ckanext/issues/tests/logic/action/test_issue.py

```diff
@@ -607,26 +607,71 @@ def test_fetch_org_editors(self):
 
 
 class TestCommentSearch(ClearOnTearDownMixin):
-    def test_search(self):
-        organization = factories.Organization()
-        dataset = factories.Dataset(owner_org=organization['id'])
+    def setup(self):
+        # Organization 1
+        self.organization = factories.Organization()
+        dataset = factories.Dataset(owner_org=self.organization['id'])
         issue = issue_factories.Issue(dataset_id=dataset['id'])
-        comment = issue_factories.IssueComment(
+
+        self.comment1 = issue_factories.IssueComment(
             issue_number=issue['number'],
             dataset_id=issue['dataset_id'],
         )
+        comment_object = IssueComment.get(self.comment1['id'])
+        comment_object.visibility = u'hidden'
+        comment_object.abuse_status = AbuseStatus.unmoderated.value
```

Comment:
I think you'd be better to do comment_object.report_abuse() instead of changing the visibility and abuse_status